test/replication-luatest/gh_6123_truncate_temp_space_test.lua

```diff
@@ -57,6 +57,10 @@ g.test_truncate_is_local_transaction = function(cg)
 
     -- Checking that replica has received the last transaction,
     -- and that replication isn't broken.
+    local vclock = cg.master:eval("return box.info.vclock")
```

Comment:
I propose to add a method `:vclock()` to server object. In `test/luatest_helpers/server.lua`. This looks like a useful method for future tests.

Reply:
There's one already in `luatest_helpers.lua` so I rebased/forcepushed